addons/mass_mailing/views/mailing_trace_views.xml

```diff
@@ -90,6 +90,9 @@
                             <field name="sent_datetime" attrs="{'invisible' : [('sent_datetime', '=', False)]}"/>
                             <field name="links_click_datetime" attrs="{'invisible' : [('links_click_datetime', '=', False)]}"/>
                             <field name="open_datetime" attrs="{'invisible' : [('open_datetime', '=', False)]}"/>
+                            <field name="failure_reason"
```

Comment:
Should we set it as readonly somehow? (On field definition?)

As it's a technical thing that is only displayed when present.